apps/studio/components/interfaces/Home/ServiceStatus.tsx

```diff
@@ -33,12 +33,12 @@ const StatusMessage = ({
   isProjectNew: boolean
   status?: ProjectServiceStatus
 }) => {
+  if (isSuccess) return 'Healthy'
```

Comment:
Shouldn't this be
```
  if (isLoading) return 'Checking status'
  if (isSuccess) {
    if (status === 'UNHEALTHY') return 'Unhealthy'
    if (status === 'COMING_UP') return 'Coming up...'
    if (status === 'ACTIVE_HEALTHY') return 'Healthy'
  }
  if (isProjectNew) return 'Coming up...'
  return 'Unable to connect'
```

Reply:
isSuccess shouldn't be mashed with status imo

i realised this is confusing but `isSuccess` here isn't from react-query, so its not indicative or a successful network request

![image](https://github.com/user-attachments/assets/c13c6b75-d518-4ea3-beba-f69de3d391c2)

Reply:
lemme refactor this to change `isSuccess` to `isHealthy`, its very easily confused haha

Reply:
okay just pushed!